### DISCUSSION TITLE
How to temporarily close typehint for a language

### DISCUSSION THREAD

**Discussion by arvyanh** [neoclide/coc.nvim#4990] [Q&A]
after I upgraded to Neovim 10.0-dev, I noticed there was some hinting for types that appeared next to the code, such as: (wasn't there in 0.9.5, or its the different build configuration of the 10.0 AppImage)

![image](https://github.com/neoclide/coc.nvim/assets/30247574/4e87fbe3-1015-4ece-87d1-a917d2edfcef)
or
![image](https://github.com/neoclide/coc.nvim/assets/30247574/006184d5-ae89-48c0-a949-16ef70924fd0)

Sometimes, the type for a CPP variable would be unreasonably long, so just asking if there is a way to temporarily disable it (or what configuration is causing that)?

**Comment by fannheyward** [comment]
Check `:h coc-inlayHint` for the configurations.
